getting-started/initialize-the-sdk.md

```diff
@@ -103,6 +103,56 @@ ACPCore.setLogLevel(ACPMobileLogLevel.debug)
 {% endtab %}
 {% endtabs %}
 
+## Registering Extensions and Starting Core
+
+All AEP extensions provide a `registerExtension` API (except Core), this API registers the extension with Core which allows the extension to dispatch and listen for events. You are required to register each of your extensions before making API calls and failing to do so will lead to undefined behavior.
+
+Once you have all your extensions registered you must call the `start` API in Core. This is required to boot up the SDK for event processing.
+
+The following code snippets demonstrate how to initialize the SDK when using the Identity, Signal, Lifecycle, and Analytics extensions.
```

Comment:
at the end of this sentence, please change from a period to a colon. (...and Analytics extensions:)